src/ui/component/channelThumbnail/view.jsx

```diff
@@ -1,15 +1,44 @@
 // @flow
 import React from 'react';
+import { parseURI } from 'lbry-redux';
+import classnames from 'classnames';
+import Gerbil from './gerbil.png';
 
 type Props = {
   thumbnail: ?string,
+  uri: string,
 };
 
 function ChannelThumbnail(props: Props) {
-  const { thumbnail } = props;
+  const { thumbnail, uri } = props;
+
+  function getColorClass() {
+    // Generate a random color class based on the first letter of the channel name
+    const { channelName } = parseURI(uri);
+    const initializer = channelName.charCodeAt(0) - 65; // will be between 0 and 57
+
+    let thumbnailColorClass = `channel-thumbnail__default--`;
+
+    switch (true) {
```

Comment:
I believe that `thumbnailColorClass += (initializer % 15)` or something close to this is equivalent to the entire `switch`

Reply:
Yep. `initializer % 4`